File: Exercicio3/exercicio3.lua
contador=0
numero=io.read("*n")
for divisor=2,numero-1 do
if numero%divisor==0 then
contador=contador+1
end
end
if contador==0 then
print(numero.." E PRIMO")
else
print(numero.." NAO E PRIMO")
end

File: Exercicio3/exercicio3.py
def primo():
    primo=int(input('Numero: '))
    for i in range(2,primo+1):
        if i!=primo:
            i=primo%i
        if i==0:
            print('Não é primo')
            break
        else:
            print('Primo')
            break
print(primo())
